auth/auth/driver/driver.py

```diff
@@ -235,119 +232,6 @@ async def delete(self):
         self.app_obj_id = None
 
 
-class DatabaseResource:
```

Comment:
For posterity: functionality to create a database will be restored in a later PR that lets us create namespaces on-demand.